mergify_engine/web/dashboard.py

```diff
@@ -43,6 +44,7 @@ async def get_stats(
         redis.get_redis_cache
     ),
 ) -> responses.Response:
+    last_seen_at = await last_seen.get(redis_cache, owner_id)
```

Comment:
can we get this using a `pipe` or something, while we get the seats count?

Reply:
We can't, `count_seats` use `SCAN` :(. We have a ticket to rethink this.